userconfig.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+echo "Configuring user session"
+
+# Make sure the user has the SWAN_projects folder
+mkdir -p $SWAN_HOME/SWAN_projects/
+
+# Setup LCG
+source $LCG_VIEW/setup.sh
+
+# Add SWAN modules path to PYTHONPATH so that it picks them
+export PYTHONPATH=$EXTRA_LIBS/modules/:$PYTHONPATH 
+
+# To prevent conflicts with older versions of Jupyter dependencies in CVMFS
```

Comment:
This we do not need anymore, right?

Reply:
So what you are saying is that old py3 stacks work, with this extra installation, except for ROOT?

Reply:
yes